tiles/src/main/java/com/protomaps/basemap/layers/Landuse.java

```diff
@@ -18,61 +19,105 @@ public static void processFeature(SourceFeature sf, FeatureCollector features, S
     if (sf.canBePolygon() && (sf.hasTag("aeroway", "aerodrome", "runway") ||
       sf.hasTag("area:aeroway", "taxiway", "runway") ||
       sf.hasTag("amenity", "hospital", "school", "kindergarten", "university", "college") ||
+      sf.hasTag("boundary", "national_park", "protected_area") ||
       sf.hasTag("landuse", "recreation_ground", "industrial", "brownfield", "railway", "cemetery", "commercial",
         "grass", "orchard", "farmland", "farmyard", "residential") ||
-      sf.hasTag("leisure", "park", "garden", "golf_course", "dog_park", "playground", "pitch") ||
+      sf.hasTag("leisure", "park", "garden", "golf_course", "dog_park", "playground", "pitch", "nature_reserve") ||
```

Comment:
Breaking change in v2 > v3: `nature_reserve` moved to landuse layer form natural layer.